CMakeLists.txt

```diff
@@ -61,9 +63,32 @@ FILE(GLOB_RECURSE DLR_INC
     "${TREELITE_SRC}/include/*.h"
 )
 
+if(USE_OPENCL)
+    message("USING OpenCL")
+    if(USE_OPENCL STREQUAL "ON")
+      find_package(OpenCL QUIET)
+      if(NOT OpenCL_FOUND)
+        message(FATAL_ERROR "OpenCL not found, please specify OpenCL location with -DUSE_OPENCL=/path/to/OpenCL")
+      endif(NOT OpenCL_FOUND)
+    else(USE_OPENCL STREQUAL "ON")
+      set(OpenCL_TOOLKIT_ROOT_DIR ${USE_OPENCL})
+      message(STATUS "Custom OPENCL_PATH=" ${OpenCL_TOOLKIT_ROOT_DIR})
+      set(OpenCL_INCLUDE_DIRS ${OpenCL_TOOLKIT_ROOT_DIR}/include)
+      set(OpenCL_LIBRARIES ${USE_OPENCL}/lib/libOpenCL.so)
+    endif(USE_OPENCL STREQUAL "ON")
+    include_directories(${OpenCL_INCLUDE_DIRS})
+    list(APPEND TVM_RUNTIME_LINKER_LIBS ${OpenCL_LIBRARIES})
+    file(GLOB RUNTIME_OPENCL_SRCS ${TVM_SRC}/src/runtime/opencl/*.cc)
+    list(APPEND DLR_SRC ${RUNTIME_OPENCL_SRCS})
+    set(USE_OPENCL OFF)
```

Comment:
Why do we turn off `USE_OPENCL` at the end?

Reply:
There seems to be a bug in dmlc/tvm's OpenCL build. Plus DLR OpenCLbuild does not rely on tvm runtime being builting with OpenCL, so turning it off to decouple the 2 USE_OPENCL switches (same name).